src/Autocomplete/doc/index.rst

```diff
@@ -159,6 +159,34 @@ After creating this class, use it in your form:
     these won't be used during the Ajax call to fetch results. Instead, include
     all options inside the custom class (``FoodAutocompleteField``).
 
+.. note::
+
+    .. versionadded:: Unreleased
```

Comment:
A few small changes here:

1) Use 2.7 for the version. That will at least probably be correct - so better than `Unreleased` :) 
2) Make the whole thing look like this:

```
.. versionadded:: 2.7

    The ability to specify the route was added in Twig Components 2.7.

The default route...
```

So, a small indented message for the `versionadded`, then go onto the description.